lib/index.ts

```diff
@@ -3,11 +3,18 @@ import * as fs from 'fs';
 import * as path from 'path';
 import * as subProcess from './sub-process';
 import { legacyPlugin } from '@snyk/cli-interface';
+import { CallGraph } from '@snyk/cli-interface/legacy/common';
+
+import * as javaCallGraphBuilder from '@snyk/java-call-graph-builder';
 import { containsJar, createPomForJar, createPomForJars, isJar } from './jar';
 import * as os from 'os';
 
+import debugLib = require('debug');
+const debug = debugLib('snyk-mvn-plugin');
+
 export interface MavenOptions extends legacyPlugin.BaseInspectOptions {
   scanAllUnmanaged?: boolean;
+  'reachable-vulns'?: boolean;
```

Comment:
How about camel case this is cli?

Reply:
I tried to do something like that in the CLI, but it's pretty ugly. The reason is that in the CLI the `Options & TestOptions` structure is being passed to the plugin, by having the `reachable-vulns` flag set up from the user, I would need to have `reachable-vulns` and `reachableVulns` at the same object.

Reply:
But then you wouldn't have this in code anywhere `'reachable-vulns'`

Reply:
In the CLI I'll have it

Reply:
@dkontorovskyy managed to find the mechanism that removes dashes from the CLI! updated the latest commit